src/components/MatomoOptOut.tsx

```diff
@@ -53,31 +31,43 @@ const MatomoOptOut: React.FC<IProps> = () => {
     localStorage.setItem(MATOMO_LS_KEY, String(!checked))
   }
   return (
-    <Container>
-      <p>
+    <Flex
+      border="1px solid"
+      borderColor="border"
+      bgColor="background"
+      borderRadius={1}
```

Comment:
```suggestion
      borderRadius="base"
```

Reply:
Why didn't we include this? we want to use the `base` radius here (`4px`), right?

Reply:
Apologies  I thought I had pushed the file that with base in the borderRadius will fix the styling right now